Sources/SmileID/Classes/DocumentVerification/Model/OrchestratedDocumentVerificationViewModel.swift

```diff
@@ -174,6 +195,19 @@ internal class IOrchestratedDocumentVerificationViewModel<T, U: JobResult>: Obse
                 receiveCompletion: { completion in
                     switch completion {
                     case .failure(let error):
+                            do {
+                                _ = try LocalStorage.saveOfflineJob(
+                                    jobId: self.jobId,
+                                    userId: self.userId,
+                                    jobType: self.jobType,
+                                    enrollment: false,
+                                    allowNewEnroll: self.allowNewEnroll,
+                                    partnerParams: self.extraPartnerParams
+                                )
+                            } catch {
+                                print("Error submitting job: \(error)")
```

Comment:
We should make this message specific to offline mode
```suggestion
                                print("Error saving job for offline mode: \(error)")
```

Reply:
updated